src/components/ConfigEditor/AzureCredentials.ts

```diff
@@ -0,0 +1,56 @@
+import { SelectableValue } from '@grafana/data';
+
+export enum AzureCloud {
+  Public = 'AzureCloud',
+  China = 'AzureChinaCloud',
+  USGovernment = 'AzureUSGovernment',
+}
+
+export const KnownAzureClouds = [
+  { value: AzureCloud.Public, label: 'Azure' },
+  { value: AzureCloud.China, label: 'Azure China' },
+  { value: AzureCloud.USGovernment, label: 'Azure US Government' },
+] as SelectableValue[];
+
+export type AzureAuthType = 'msi' | 'clientsecret' | 'clientsecret-obo';
```

Comment:
nit: This can be an `enum` too

Reply:
it's functionally the same but already defined this way in other datasources.